core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -350,4 +376,23 @@ protected synchronized InetSocketAddress getAddress() {
     }
     throw new IOException("Failed after " + retry + " retries.");
   }
+
+  private class TransportOpenCallable implements Callable<Void> {
```

Comment:
Would it be possible to implement this as an anonymous class? It seems fairly one-time use.